source/docs/prismic/routable-types.md

```diff
@@ -0,0 +1,220 @@
++      graphQLType: "Album",    // <-- GraphQL type created above
++      contentTransformOptions,
++      isSitemapable: false,
++      postTransformer: (url, document) => { // optional function postTransformer
++        if(document.isPublished) {  // possible usecase
++          return document;
++        }
++      }
++    });
+
+    return {};
+  },
+};
+```
+
+Note the `postTransformer` above is optional. It is a function that will be called after the transformation is done using `contentTransformOptions`. it is given the current URL being resolved and the transformed document. It can be used if you have some custom logic to apply to the transformed document or if you want to supress a document from showing using some custom logic (returning a falsey value).
+
+## Map GraphQL type to a component
+
+To map a GraphQL type to React component, create a file called `moduleRoutes.js` at the root of the `web` directory and add the following to it:
+
+```js
+import React from "react";
+
+export const dispatchedRoutes = {
+  Album: ({ loading, matched }) => {
+    if (loading) {
+      return <div>Loading...</div>;
+    }
+    if (!matched) {
+      return <div>No Album</div>;
+    }
+    const { title, artist, release_date } = matched;
+    return (
+      <div>
+        <h3>Album:</h3>
+        {title}
+        <br />
+        {artist}
+        <br />
+        {release_date?.toString()}
+        <br />
+      </div>
+    );
+  },
+};
+```
+
+For more details please refer to [Add the mapping between the type and the page component](/docs/advanced/theme/route-dispatcher.html#Add-the-mapping-between-the-type-and-the-page-component) in the [route dispatcher](/docs/advanced/theme/route-dispatcher.html).
+
+Now an instance of the custom Prismic type with the `url` field set to `my-custom-url` will be accessible at `https://<website_domain>/my-custom-url` (or `https://<website_domain>/<shop_name>/my-custom-url` if in a multi shop environment)
+
+## Add a routable custom type to the sitemap
+
+Optionally to add the custom Prismic type to the sitemap use the following steps:
+
+1. implement the Sitemapable interface on the GraphQL type, and resolver.
+
+```diff
+// schema.gql
+-type Album implements Routable {
++type Album implements Routable & Sitemapable {
+  url: String # the newly added field
+  artist: String
+  title: String
+  release_date: Date
+  path: String # for the routable interface
++  priority: Float
++  seoImages: [SitemapImage]
++  lastmod: String
++  changefreq: String
+}
+```
+
+```diff
+// resolvers.js
+export default {
+  Query: {
+    ...
+  },
+  Album: {
+    path: async (content, _, { loaders }) => {
+      const baseUrl = (await loaders.Shop.getCurrentShop()).baseUrl,
+      return `{baseUrl}/${content.url}`;
+    },
++    priority: () => 1,
++    seoImages: (content) => [],
++    lastmod: () => "2020-01-01T12:00:00.000Z",
++    changefreq: () => "daily",
+  },
+};
+```
+
+2. Enable sitemap in the module definition file:
```

Comment:
wrong number in the resulting page https://deploy-preview-208--heuristic-almeida-1a1f35.netlify.app/docs/prismic/routable-types.html

I think you need to get rid of the empty lines and add an indeed level to code blocks